### ISSUE TITLE
Inconsistency in doc for kube-apiserver-etcd-client Certificate `O=system:masters` requirement

### ISSUE THREAD

**Issue by nnlkcncff** [kubernetes/website#42724]
> https://kubernetes.io/docs/setup/best-practices/certificates/#all-certificates

According to the documentation, the `kube-apiserver-etcd-client` certificate requires `O=system:masters`, but how can it be needed if etcd doesn't take this into account?

Perhaps `O=system:masters` should be removed  for the `kube-apiserver-etcd-client` certificate and then I can create an issue on the kubeadm repo like like [the one](https://github.com/kubernetes/kubeadm/issues/2915) I created earlier.

**Comment by dipesh-rawat** [comment]
_Page related to issue:_ https://kubernetes.io/docs/setup/best-practices/certificates/
/language en

**Comment by dipesh-rawat** [comment]
/retitle Inconsistency in doc for kube-apiserver-etcd-client Certificate `O=system:masters` requirement
/sig cluster-lifecycle

**Comment by neolit123** [comment]
> Perhaps O=system:masters should be removed for the kube-apiserver-etcd-client certificate and then I can create an issue on the kubeadm repo like like the one I created earlier.

+1
PRs welcome
thanks

**Comment by nnlkcncff** [comment]
@neolit123, sorry but I don't understand what do you mean — just agreement or confirmation about what should be done? :)
You seem to confirm that `O=system:masters` is indeed not required in the `kube-apiserver-etcd-client certificate` certificate, but I'm not sure because there are still no updates in the issue and the documentation is still not changed. So, I can't figure out if I should create an issue in the kubeadm repository.

**Comment by neolit123** [comment]
yes, you can log the issue in k/kubeadm.
in terms of updating the docs, i said "prs welcome", thus if you l'd like to help us you can send the docs change / pull request.

**Comment by nnlkcncff** [comment]
@neolit123, thanks for your clarification. You're not the first to respond with "PR welcome," but I assumed it wasn't directed at me. Ok, I guessed, but I wasn’t sure.

I don't know why I want to say the following, but I want :) So...

I admit, I didn’t want to do PR and sort out the processes adopted in this project, so I just wrote down and saved minor changes for later. But I have already accumulated about 50 of them and I plan to deal with them as soon as I finish studying the documentation — I’m already close. I haven't been paying much attention, but today I realized how much technical debt I've accumulated to myself.
